internal/app/connectconformance/testsuites/data/server_unexpected.yaml

```diff
@@ -21,7 +18,14 @@ testCases:
             value: [ "image/jpeg" ]
         unary:
           binary: "12345678"
+    otherAllowedErrorCodes:
+      # Not actually specified what error code to use, but only
+      # internal and unknown really make any sense. There may be
+      # an argument for unimplemented, too.
+      - CODE_INTERNAL
+      - CODE_UNIMPLEMENTED
     expectedResponse:
-      httpStatusCode: 415
+      # Ideally, the server would return a 415. But it's not actually specified :/
+      # httpStatusCode: 415
       error:
-        code: CODE_UNKNOWN
\ No newline at end of file
+        code: CODE_UNKNOWN
```

Comment:
IIRC the Connect protocol is very particular about this scenario - should we separate the gRPC and Connect test expectations?

Reply:
Sure. I was actually thinking about that this morning, too.